src/binary-reversal/index.js

```diff
@@ -3,6 +3,22 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  //Converting value to a binary string
+  let binaryValue = parseInt(value, 10).toString(2);
+
+  // Adding zero if to make up length to 8
+  while (binaryValue.length < 8) {
+    binaryValue = "0" + binaryValue;
+  }
```

Comment:
Same as `binaryValue.padStart()`

Reply:
Didn't even know that was a method. Noted.

Thanks